python/ray/autoscaler/_private/constants.py

```diff
@@ -42,6 +42,10 @@ def env_integer(key, default):
 AUTOSCALER_HEARTBEAT_TIMEOUT_S = env_integer("AUTOSCALER_HEARTBEAT_TIMEOUT_S",
                                              30)
 
+# The maximum number of nodes (including failed nodes) that the autoscaler will
+# track for logging purposes.
+AUTOSCALER_MAX_NODES_TRACKED = 1500
```

Comment:
```suggestion
AUTOSCALER_MAX_NODES_TRACKED = 20
```

1500 will blow up in the UI, let's start with say 20 for now

Reply:
This includes live nodes, so we should probably make sure it's at least a little larger than our scalability envelope

Reply:
How about 300 for now?